config/nginx.conf

```diff
@@ -9,19 +9,4 @@ server {
         uwsgi_pass_request_headers on;
         uwsgi_pass_request_body on;
     }
-
-    location @webpack {
-        proxy_pass http://watch:8078;
-    }
-
```

Comment:
This can be removed if all assets that would be served by webpack are addressed directly in the HTML.